string device_path
string[] channels
std_msgs/Int32MultiArray buffer
bool cyclic
---
bool success
string message
